platform/Cmd.roc

```diff
@@ -164,8 +162,8 @@ status = \@Cmd cmd ->
 exec : Str, List Str -> Task {} [CmdError Err]
 exec = \program, arguments ->
 
-    @Cmd cmd = new program |> args arguments
+    (@Cmd cmd) = new program |> args arguments
```

Comment:
Was this a formatter change, or do we need this syntactically now?

Reply:
We do not need the parens for this file to `check`. The formatter adds these both in the latest and previous nightly.